jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -48,6 +51,9 @@ def buildExclusions = [
   [ /^cross-compiler-armv[67]-gcc-4.9/, anyType,   lt(10)  ],
   [ /^cross-compiler-armv[67]-gcc-4.9/, anyType,   gte(12) ],
   [ /^cross-compiler-armv[67]-gcc-6/, anyType,     lt(12)  ],
+  [ /^cross-compiler-armv[67]-gcc-6/, anyType,     gte(14) ],
+  [ /^cross-compiler-armv[67]-gcc-8/, anyType,     lt(14)  ],
+  [ /^ubuntu1604-arm64/,              anyType,     gte(14) ],
```

Comment:
Unrelated to devtools8, but related to 14.x requirements, see comment https://github.com/nodejs/build/issues/2242#issuecomment-611190279

I'm not sure if we are comfortable with this, but there is a case to be made that it should be excluded for gte(12.x), cc: @nodejs/releasers , because it went EOL on the 2019-april-25th, https://ubuntustudio.org/2019/04/ubuntu-studio-16-04-lts-reaches-end-of-life-eol/, 2 days after `(tag: v12.0.0) 2019-04-23, Version 12.0.0 (Current)`, so we should have excluded it from test then.

Reply:
ack 👍